app/src/main/res/layout/fragment_add_product_nutrition_facts.xml

```diff
@@ -168,9 +168,10 @@
                 android:layout_marginStart="@dimen/activity_horizontal_margin"
                 android:layout_marginTop="@dimen/spacing_small"
                 android:background="@drawable/bg_edittext"
+                android:digits="0123456789.~&gt;&lt;"
                 android:gravity="center_vertical"
                 android:hint="@string/nutrition_energy"
-                android:inputType="numberDecimal"
+                android:inputType="textVisiblePassword"
```

Comment:
?

Reply:
number input type shows a numeric keyboard which doesn't allow special characters like `< > or ~` to be inputted. Since we need these along with the numeric input I changed the input type to `textVisiblePassword` which displays a keyboard with a numeric row on the top of the alphabets.

Reply:
weird SDK naming. :-)